doc/CMakeLists.txt

```diff
@@ -191,6 +195,8 @@ if(DOXYGEN_FOUND)
   list(APPEND CMAKE_DOXYGEN_HTML_FILES "${CMAKE_CURRENT_SOURCE_DIR}/tutorial-utils.js")
   string(REPLACE ";" " \\\n" CMAKE_DOXYGEN_HTML_FILES "${CMAKE_DOXYGEN_HTML_FILES}")
 
+  set(OPENCV_MATHJAX_RELPATH "https://cdn.jsdelivr.net/npm/mathjax@3.0.1" CACHE STRING "URI to a MathJax installation")
```

Comment:
> `if(DOXYGEN_FOUND)`
> `OPENCV_MATHJAX_RELPATH`

This variable is used in Javadoc too which could be build without Doxygen.

Moving from the root CMake script is wrong.

P.S. History is broken due to removal.